pkg/kubelet/remote/remote_image.go

```diff
@@ -32,12 +32,13 @@ import (
 
 // RemoteImageService is a gRPC implementation of internalapi.ImageManagerService.
 type RemoteImageService struct {
-	timeout     time.Duration
-	imageClient runtimeapi.ImageServiceClient
+	timeout                   time.Duration
```

Comment:
Do you have a sense of how difficult it would be to give `timeout` more specific name? It looks like `timeout` is actually the `connection` timeout - however, that isn't particularly clear without reading the source code. I worry this generic name will become even more confusing with the addition of `imagePullProgressDeadline`.

Reply:
make sense, change to connectionTimeout